File: abc/abc285/a.java
// https://atcoder.jp/contests/abc285/submissions/41532943

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    if(b / 2 == a) System.out.println("Yes");
    else System.out.println("No");
  }
}

File: abc/abc285/a.cs
// https://atcoder.jp/contests/abc285/submissions/46509367

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    int b = int.Parse(t[1]);

    if(b / 2 == a)
    {
      Console.Write("Yes");
    }
    else
    {
      Console.Write("No");
    }
  }
}

